oauthenticator/oauth2.py

```diff
@@ -223,10 +223,11 @@ class OAuthenticator(Authenticator):
     Subclasses must override:
 
     login_service (string identifying the service provider)
-    login_handler (likely a subclass of OAuthLoginHandler)
     authenticate (method takes one arg - the request handler handling the oauth callback)
     """
 
+    login_handler = OAuthLoginHandler
+
```

Comment:
Another line further down redefines `login_handler`,  nullifying this one. After removing that line, I did a quick test (using Globus) and everything looked good.

Reply:
Good catch, fixed!